communication/icons/hack/convert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+HEIGHT=249
```

Comment:
Comment on the required aspect ratio?  E.g. if we just did 256x256 would it be so bad (to have just one param) ?